src/types/index.ts

```diff
@@ -167,6 +167,13 @@ export interface Tenants {
     gcm_account_id?: string | null;
 }
 
+export interface TenantPaymentDetails {
+    gcm_account_id: string | null;
+    payment_provider: TenantPaymentProviders;
+    tenant: string;
+    trial_start: string;
+}
```

Comment:
We don't need all the details so saving a tiny bit of bits by reducing the query.